src/include/miopen/batch_norm.hpp

```diff
@@ -66,10 +67,14 @@ void bnBwdTrainSelectSingle(const Handle& handle,
                             double epsilon,
                             ConstData_t savedMean,
                             ConstData_t savedInvVariance,
-                            float inhw);
+                            float nhw,
```

Comment:
I see you are calculating inverse nhw in OCL code. is it possible to send it inhw directly?

Reply:
In backwards we use both 1/nhw and nhw. I suppose I could send the both in. Maybe in another PR.